src/Digdir.Domain.Dialogporten.Infrastructure/InfrastructureExtensions.cs

```diff
@@ -212,6 +216,23 @@ public static IServiceCollection AddInfrastructure(this IServiceCollection servi
         return services;
     }
 
+    private static IServiceCollection AddCustomHealthChecks(this IServiceCollection services)
+    {
+        services.AddHttpClient("HealthCheckClient")
+            .ConfigureHttpClient((services, client) =>
+            {
+                client.Timeout = TimeSpan.FromSeconds(5);
+            });
+
+        services.AddHealthChecks()
```

Comment:
Yeah om jeg tolker dette riktig: 

This operation is idempotent - multiple invocations will still only result in a single HealthCheckService instance in the IServiceCollection. It can be invoked multiple times in order to get access to the IHealthChecksBuilder in multiple places.

Ser ut til å fungere og!